src/driver/mysql/MysqlDriver.ts

```diff
@@ -829,7 +834,12 @@ export class MysqlDriver implements Driver {
             debug: options.debug,
             trace: options.trace,
             multipleStatements: options.multipleStatements,
-            flags: options.flags
+            flags: options.flags,
+            ...(
+                this.version === 1
+                    ? { acquireTimeout: options.acquireTimeout } 
+                    : {}
+                )
```

Comment:
i think this statement is particularly hard to read.. but that is just my opinion